### ISSUE TITLE
Introduce a "failsafe" flag for risky private operations

### ISSUE THREAD

**Issue by zevaverbach** [LeeChunHao2000/ftx-api-wrapper-python3#12]
For any private operations which are "risky" -- meaning that they'll actually do something with assets when they're called, like buy/sell assets -- what do you think about an `im_sure` flag which defaults to `False`? And it could maybe raise an exception when it's set to `False` saying "Please set `im_sure` to `True` if you want this transaction to go through."

**Comment by zevaverbach** [comment]
That's a cool idea! Although I also respect your original choice of not implementing withdrawals. The main strength of this is for algorithmic trading, right?

For that reason, I'm rethinking my suggestion to add failsafes. What do you think?

**Comment by LeeChunHao2000** [comment]
Yes, I think failsafes is a good idea that users can decide how much they wanna risk.

**Comment by zevaverbach** [comment]
Can you give an example of how that API might work?

**Comment by LeeChunHao2000** [comment]
which one?

**Comment by zevaverbach** [comment]
withdrawal with an OTP

**Comment by LeeChunHao2000** [comment]
We can find the params [here](https://docs.ftx.com/#request-withdrawal), the last param `code` is 2fa code, we can get it via this [lib](https://github.com/pyauth/pyotp).

```
totp = pyotp.TOTP('secret get from FTX 2fa setting')
totp.now() # => '492039'
```

**Comment by zevaverbach** [comment]
but how would you integrate it with withdrawal methods?

**Comment by LeeChunHao2000** [comment]
Sorry for late, I will integrate this part after I finish final exam.
